packages/@aws-cdk/aws-apigateway/lib/restapi.ts

```diff
@@ -156,6 +156,11 @@ export interface RestApiProps extends ResourceOptions {
    * @default - no domain name is defined, use `addDomainName` or directly define a `DomainName`.
    */
   readonly domainName?: DomainNameOptions;
+
+  /**
+   * Export name for the CfnOutput containing the API endpoint
+   */
```

Comment:
"@default" is required and should say that an output will be created but without an export.